src/screens/DiscoveryScreenV2/index.js

```diff
@@ -225,11 +326,11 @@ const styles = StyleSheet.create({
     paddingHorizontal: 24,
     paddingVertical: 4,
     fontFamily: fonts.inter[600]
-  }
+  },
+  headerTitle: {fontSize: 16, fontFamily: fonts.inter[600], textAlign: 'center'}
 });
 
 export default withInteractionsManagedNoStatusBar(DiscoveryScreenV2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export statement has been updated to remove the commented out `React.memo`. If the component doesn't need to be memoized, this is a good change. However, if the component's props are complex and it re-renders often, consider using `React.memo` for performance optimization.


<!-- This is an auto-generated comment by OSS CodeRabbit -->